__tests__/fetching_conversation_starters.test.js

```diff
@@ -15,9 +17,5 @@ test('fetches the source with the matching tag', () => {
         },
         tags: ['tag']
     });
-
-    bot.notify('test-channel', new Date(0));
-    bot.reviveChannels(new Date(15 * 1000));
-
-    expect(message).toBe('expected');
+    expect(bot.chooseSourceFor(bot.channels.get('test-channel'))).toBe(correctSource);
```

Comment:
Oh hmm that's kinda testing the internals

Reply:
It is. If you have thoughts on a better way to do it, I'd love it